tests/semantic/agg_nested/agg_nested.err

```diff
@@ -0,0 +1,4 @@
+Error: Unsupported nested aggregate:  in file agg_nested.dl at line 19
```

Comment:
I think it should be `Error: Unsupported nested aggregate in file...` instead of `Error: Unsupported nested aggregate:  in file...`